data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd

```diff
@@ -108,16 +113,16 @@ battle_trap_clean %>% glimpse
 - sub_week: is related to efficiency and included in efficiency table
 - sample_weight: same as cone variable
 - flow_start_meter, flow_end_meter, flow_set_time, velocity, turbidity,
-weather_code, lunar_phase, diel included in environmental table
+weather_code, lunar_phase, diel, river_left_depth, river_center_depth, river_right_depth included in environmental table
```

Comment:
I agree that it is fine to remove these but they are not all included in the environmental table. I think that is okay but we should make the comment consistent. Same for other tribs where these were removed.

Reply:
good catch. i revised comments for accuracy. next step is to revisit the standard formats. during this process i will make sure that the depth variables are included in the environmental table or clearly documented why they are left out.